swarm_copy/tools/resolve_entities_tool.py

```diff
@@ -0,0 +1,154 @@
+"""Tool to resolve the brain region from natural english to a KG ID."""
+
+import logging
+from typing import ClassVar
+
+from langchain_core.tools import ToolException
+from pydantic import BaseModel, ConfigDict, Field
+
+from neuroagent.resolving import resolve_query
```

Comment:
This function should not be imported from neuroagent, since we want to ditch it.